spec/features/merchants/items/index_spec.rb

```diff
@@ -58,4 +58,26 @@
     expect(page).to have_link("#{@item_2.name}", href: merchant_item_path(@merchant_1.id, @item_2.id))
     expect(page).to have_link("#{@item_3.name}", href: merchant_item_path(@merchant_1.id, @item_3.id))
   end
+
+  describe 'item disable/enable button' do 
+    scenario 'visitor sees a button to disable that item' do 
+      within "#enabled_item-#{@item_1.name}" do
+        expect(page).to have_button("Disable")
+        click_button "Disable"
+      end
+      
+      expect(page).to have_content("Enable")
+      expect(current_path).to eq(merchant_items_path(@merchant_1.id))
+    end 
+
+    scenario 'visitor sees a button to disable that item' do 
+      within "#disabled_item-#{@item_3.name}" do
+        expect(page).to have_button("Enable")
+        click_button "Enable"
+      end
+      
+      expect(page).to have_content("Disable")
```

Comment:
will we also be expecting have_button("Disable") rather than just content?

Reply:
Yes, thank you!